packages/bot-skeleton/src/scratch/hooks/colours.js

```diff
@@ -1,64 +1,71 @@
-            colourTertiary: '#151717',
-        };
-        Blockly.Colours.Base = {
-            colour: '#C2C2C2',
-            colourSecondary: '#0e0e0e',
-            colourTertiary: '#0e0e0e',
-        };
+const darkMode = () => {
+    const workspace = Blockly;
+    workspace.Colours.RootBlock = {
+        colour: '#183046',
+        colourSecondary: '#F2F3F5',
+        colourTertiary: '#151717',
+    };
+    workspace.Colours.Base = {
+        colour: '#C2C2C2',
+        colourSecondary: '#0e0e0e',
+        colourTertiary: '#0e0e0e',
+    };
 
-        Blockly.Colours.Special1 = {
-            colour: '#C2C2C2',
-            colourSecondary: '#0e0e0e',
-            colourTertiary: '#6d7278',
-        };
+    workspace.Colours.Special1 = {
+        colour: '#C2C2C2',
+        colourSecondary: '#0e0e0e',
+        colourTertiary: '#6d7278',
+    };
 
-        Blockly.Colours.Special2 = {
-            colour: '#C2C2C2',
-            colourSecondary: '#0e0e0e',
-            colourTertiary: '#D27954',
-        };
+    workspace.Colours.Special2 = {
+        colour: '#C2C2C2',
+        colourSecondary: '#0e0e0e',
+        colourTertiary: '#D27954',
+    };
 
-        Blockly.Colours.Special3 = {
-            colour: '#C2C2C2',
-            colourSecondary: '#0e0e0e',
-            colourTertiary: '#D27954',
-        };
-    } else {
-        Blockly.Colours.RootBlock = {
-            colour: '#064e72',
-            colourSecondary: '#064e72',
-            colourTertiary: '#6d7278',
-        };
+    workspace.Colours.Special3 = {
+        colour: '#C2C2C2',
+        colourSecondary: '#0e0e0e',
+        colourTertiary: '#D27954',
+    };
+};
+
+const lightMode = () => {
+    const workspace = Blockly;
+    workspace.Colours.RootBlock = {
+        colour: '#064e72',
+        colourSecondary: '#064e72',
+        colourTertiary: '#6d7278',
+    };
 
-        Blockly.Colours.Base = {
-            colour: '#e5e5e5',
-            colourSecondary: '#ffffff',
-            colourTertiary: '#6d7278',
-        };
+    workspace.Colours.Base = {
+        colour: '#e5e5e5',
+        colourSecondary: '#ffffff',
+        colourTertiary: '#6d7278',
+    };
 
-        Blockly.Colours.Special1 = {
-            colour: '#e5e5e5',
-            colourSecondary: '#ffffff',
-            colourTertiary: '#6d7278',
-        };
+    workspace.Colours.Special1 = {
+        colour: '#e5e5e5',
+        colourSecondary: '#ffffff',
+        colourTertiary: '#6d7278',
+    };
 
-        Blockly.Colours.Special2 = {
-            colour: '#e5e5e5',
-            colourSecondary: '#ffffff',
-            colourTertiary: '#6d7278',
-        };
+    workspace.Colours.Special2 = {
+        colour: '#e5e5e5',
+        colourSecondary: '#ffffff',
+        colourTertiary: '#6d7278',
+    };
 
-        Blockly.Colours.Special3 = {
-            colour: '#e5e5e5',
-            colourSecondary: '#ffffff',
-            colourTertiary: '#6d7278',
-        };
+    Blockly.Colours.Special3 = {
```

Comment:
shouldn't this be `workspace.Colours.Special3`?

Reply:
yes will change this 👍